src/Text/Pandoc/Writers/FB2.hs

```diff
@@ -325,9 +317,10 @@ blockToXml (Para [Image atr alt (src,'f':'i':'g':':':tit)]) =
 blockToXml (Para ss) = (list . el "p") <$> cMapM toXml ss
 blockToXml (CodeBlock _ s) = return . spaceBeforeAfter .
                              map (el "p" . el "code") . lines $ s
-blockToXml b@(RawBlock _ _) = do
-  report $ BlockNotRendered b
-  return []
+blockToXml (RawBlock f str) = do
+  if f == Format "xml"
```

Comment:
This should say `Format "fb2"`

Reply:
also, indicate in your commit message that this change is being made (since it's independent of the section identifiers one).  Ideally this would be a separate commit, since it's a logically separate change, but if it's hard to separate out, don't worry about it.

Reply:
Ok, I will make a separate commit.
Why it should be "fb2"? For example in epub with raw block we specify "html" format.

Reply:
@KetzerX the `Format` should be one of the strings you can use in pandoc in the `--to` flag. The EPUB writer is somewhat special in that it just runs the HTML writer and then zips the output, that's why it's `html` there sometimes instead of `epub`.

Reply:
Thanks. Done.